### DISCUSSION TITLE
How to properly use motion sensors.

### DISCUSSION THREAD

**Discussion by lockeandev** [tsightler/ring-mqtt#669] [Support Q&A] [answered]
I've had the hass add-on version running for over a year and have never tried to use the motion sensor to trigger any automations. I'm trying that now, and am finding that none of my cameras motion sensor is reporting correctly in Home Assistant. The sensor just always have a "clear" status.

Can I not use the cameras motion sensors to trigger automations?

All of my ring cameras are wired flood cameras.

**Comment by tsightler** [comment, marked as answer]
Most like you are experiencing the issue documented in the [release notes for v5.3.0](https://github.com/tsightler/ring-mqtt/blob/main/docs/CHANGELOG.md#v530).

TL;DR Ring made changes to their API that broke the method ring-client-api was using the register for push notifications.  This was fixed back in June, but, if the problem had already occurred, the only fix was to remove all old authorized devices and to re-authenticate.  After that, notifications should work again.

**Reply by lockeandev** [reply]
Perfect that worked. Thanks for all your work on this!
